README.md

```diff
@@ -1,6 +1,13 @@
 README for Mbed TLS
 ===================
 
+Mbed TLS makes it trivially easy for developers to include cryptographic and SSL/TLS capabilities in their (embedded) products, facilitating this functionality with a minimal coding footprint.
```

Comment:
This is marketing copy and it reads as a little silly in a project `README`. I'd drop the word 'trivially' and the word 'embedded', and the phrase 'facilitating this functionality with a minimal coding footprint'.

Reply:
I'm taking Gilles's text here. I just want a project description of some sort.